src/themes/oneui/base-variables/_colors.scss

```diff
@@ -67,6 +67,6 @@ $color-bad: #f42534 !default;
     --border-color-light: var(--color-neutral-25);
     --border-color-strong: var(--color-neutral-40);
 
-    --color-hover-background: #{mix($color-background, $color-brand, 90)};
-    --color-highlight-background: #{mix($color-background, $color-brand, 90)};
+    --color-highlight-background: var(--color-info-10);
```

Comment:
why have you removed `mix` function here?

Reply:
@antipin I replaced it with a slightly brighter color so the item would have visible difference between `highlighted` and `selected` style options, but I'm going to discuss this part in details with David so let it hang here for a while please

Reply:
@antipin can we mark it as resolved?

Reply:
@igorarkhipenko 👍